test/common.test.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { SfdcUrl, SfdxProject } from '@salesforce/core';
+import sinon = require('sinon');
+import { expect } from '@salesforce/command/lib/test';
+import { restoreContext, testSetup } from '@salesforce/core/lib/testSetup';
+import { Common } from '../src/common';
+
+describe('common unit tests', () => {
+  const sandbox = sinon.createSandbox();
+  const $$ = testSetup();
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars,@typescript-eslint/no-explicit-any
+  let projectPath: any;
```

Comment:
why is this `any`? shouldn't that be a string?

Reply:
changing to string